corehq/apps/users/forms.py

```diff
@@ -371,6 +371,7 @@ def __init__(self, project, user_id, **kwargs):
     def clean_new_password1(self):
         if self.project.strong_mobile_passwords:
             return clean_password(self.cleaned_data.get('new_password1'))
+        return self.clean_data.get('new_password1')
```

Comment:
It was fixed in a later commit https://github.com/dimagi/commcare-hq/pull/12615/commits/6780ce54d5a6abf70c3dff651ce5cf4b244e431b